### DISCUSSION TITLE
Decoupled CMS - deploy to docker

### DISCUSSION THREAD

**Discussion by scottiemc7** [OrchardCMS/OrchardCore#8767] [Q&A] [answered]
Hi. Thank you for this most excellent project! I'm following along with the [how to create a decoupled CMS site tutorial](https://docs.orchardcore.net/en/dev/docs/guides/decoupled-cms), and it all seems to be working well. When I add a custom module to the application I start having problems. I can see every Razor page view in the module when I'm debugging from the IDE (Rider in this case), but when I build and publish the app using docker those views can no longer be found. Has anyone else run across this issue? The test app I'm using [is here](https://github.com/scottiemc7/orchardcore-decoupled-test). Thanks for your help.

**Comment by Skrypt** [comment, marked as answer]
https://github.com/scottiemc7/orchardcore-decoupled-test/blob/f1c1c7407d7119df2a994d5d57d1c9f7d7b78201/AnotherTest/AnotherTest.csproj#L6

https://github.com/scottiemc7/orchardcore-decoupled-test/blob/f1c1c7407d7119df2a994d5d57d1c9f7d7b78201/MyFirstMod/MyFirstMod.csproj#L6

`<RazorCompileOnPublish>false</RazorCompileOnPublish>`

Remove this

**Reply by scottiemc7** [reply]
Oops, I threw that in as a Hail Mary attempt after [seeing this](https://github.com/OrchardCMS/OrchardCore/issues/3553#issuecomment-488228557). I removed it from both projects, and still got the same result 😞

**Reply by Skrypt** [reply]
The issue is with your Dockerfile. You are building only your AnotherTest project.
See, you are copying only the AnotherTest project folder.

If you want to follow how we do this. Put both of these projects inside a /src folder and then do

COPY ./src /app
RUN dotnet publish /app/OrchardCore.Cms.Web -c Release -o ./build/release --framework net5.0

**Reply by scottiemc7** [reply]
Oh you're right! I had assumed that since I could see my module in OC that I was publishing the app properly. Duh. Thanks for the help!
